coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -23,16 +23,21 @@ class flat_tax_on_salary(Variable):
 
  and let's assume we have in one of our parameter files the value of the rate for the past couple of years:
 
-```xml
-<NODE code="taxes">
-    <NODE code='salary'>
-      <CODE code="rate" description="Rate for the flat tax on salaries">
-        <VALUE deb="2016-01-01" fuzzy="true" valeur="0.25" />
-        <VALUE deb="2015-01-01" fin="2015-12-31" valeur="0.20" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="0.22" />
-      </CODE>
-    </NODE>
-</NODE>
+```yaml
+type: node
+taxes:
+  type: node
+  salary:
+    type: node
+    rate:
+      description: Rate for the flat tax on salaries
+      values:
+        '2014-01-01':
+          value: 0.22
+        '2015-01-01':
+          value: 0.20
+        '2016-01-01':
+          value: 0.25
```

Comment:
Is it normal that dates are in growing order in the input example and the other way around in the output ?